bin/main.rb

```diff
@@ -1,3 +1,35 @@
 #!/usr/bin/env ruby
 
-puts 'Hello World!'
+require 'pry'
+board = [' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ']
+def display_board(board)
+  puts " #{board[0]} | #{board[1]} | #{board[2]} "
+  puts '-----------'
+  puts " #{board[3]} | #{board[4]} | #{board[5]} "
+  puts '-----------'
+  puts " #{board[6]} | #{board[7]} | #{board[8]} "
+end
+
+def draw_welcome
+  puts '************************************************************'
+  puts '*                        TIC-TAC-TOE                       *'
+  puts '************************************************************'
+  puts "      by the Awesome Dev:  David Bassey\n\n"
+  puts '                                                 '
+  puts '-------------GAME INSTRUCTIONS---------------'
+  puts 'The first player takes X and the second player takes O'
+  puts 'Choose numbers from 1 to 9 to select desired cell'
+  puts 'No duplicate numbers are allowed'
+  puts '--------------------------------'
+  puts 'Player 1: What is your name?:'
+  player_one = gets.chomp
+  puts 'Player 2: What is your name?:'
+  player_two = gets.chomp
+  puts '************************************************************'
+  puts "#{player_one} gets to pick X , while #{player_two} gets to pick O"
+  puts '************************************************************'
+  puts 'Game Start!'
+end
+
+p draw_welcome
+p display_board(board)
```

Comment:
- [ ] A loop is needed in this game (recommended is while but anything else will do) (OdinProject hint)

 - [ ] Inside the loop, you should call actions repeated for each move

  - [ ] A loop needs to be terminated at some point (after a winning move or draw to be more specific). It can be done with variable:
```ruby
game_on = true

# initial instructions for player

while game_on
  # loop for each move

  if winner # and/or draw (the exact condition in this milestone is not important)
    game_on = false
  end
end
```